vuejs/src/org/jetbrains/vuejs/intentions/expandVModel/VueExpandVModelIntention.kt

```diff
@@ -16,7 +16,8 @@ class VueExpandVModelIntention : JavaScriptIntention() {
     private val validModifiers = setOf("lazy", "number", "trim")
 
     override fun isAvailable(project: Project, editor: Editor?, element: PsiElement): Boolean {
-        return element.node.elementType == XmlElementType.XML_NAME &&
+        return org.jetbrains.vuejs.index.isVueContext(element) &&
```

Comment:
The vue context check might be an expensive operation as it needs to hit the cache and possibly filesystem. The `isAvailable` method is run on every PsiElement in the file, so it is better to move the context either just before `isValid` call or maybe better after. I think it would be good to add an import for `isVueContext`.

Reply:
I reordered it as last